website/project/model.py

```diff
@@ -3815,6 +3815,8 @@ class DraftRegistration(StoredObject):
 
     approval = fields.ForeignField('draftregistrationapproval', default=None)
 
+    status_logs = fields.DictionaryField(default=dict)
```

Comment:
This hinges on the answer to https://github.com/CenterForOpenScience/osf.io/pull/5644#discussion_r62885659